From 1212be6727cfaa14760f0130cdea635c38fb523b Mon Sep 17 00:00:00 2001
From: Manoj Srivastava <srivasta@golden-gryphon.com>
Date: Wed, 14 May 2014 23:46:05 -0700
Subject: [PATCH 3/3] [debian-cmake-fixes]: Change install paths to FHS
 compliant games

So the binary lives in /usr/games/tome, instead of /usr/bin/tome

Signed-off-by: Manoj Srivastava <srivasta@golden-gryphon.com>
---
 CMakeLists.txt     |  2 +-
 src/CMakeLists.txt | 10 +++++-----
 2 files changed, 6 insertions(+), 6 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 91512a6..72cd53e 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -94,7 +94,7 @@ endif(WIN32)
 # Set the path for loading the library bits.
 #
 IF(SYSTEM_INSTALL)
-  SET(DEFAULT_PATH "${CMAKE_INSTALL_PREFIX}/lib/tome")
+  SET(DEFAULT_PATH "/var/games/tome")
 ELSE()
   SET(DEFAULT_PATH "./lib")
 ENDIF()
diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index e2e6264..932fef4 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -4,9 +4,9 @@ ADD_SUBDIRECTORY(lua)
 SET(SRCS
   main-gcu.c main-x11.c main-xaw.c main-sdl.c main-gtk2.c
   z-rand.c z-util.c z-form.c z-virt.c z-term.c
-  variable.c tables.c plots.c util.c cave.c dungeon.c 
-  melee1.c melee2.c modules.c 
-  object1.c object2.c randart.c squeltch.c traps.c 
+  variable.c tables.c plots.c util.c cave.c dungeon.c
+  melee1.c melee2.c modules.c
+  object1.c object2.c randart.c squeltch.c traps.c
   monster1.c monster2.c monster3.c
   xtra1.c xtra2.c skills.c powers.c gods.c
   spells1.c spells2.c
@@ -15,7 +15,7 @@ SET(SRCS
   help.c
   generate.c gen_maze.c gen_evol.c wild.c levels.c store.c bldg.c
   cmovie.c
-  wizard2.c init2.c birth.c wizard1.c init1.c main.c 
+  wizard2.c init2.c birth.c wizard1.c init1.c main.c
   # Lua bits:
   lua_bind.c script.c w_mnster.c w_player.c w_play_c.c w_z_pack.c
   w_obj.c w_util.c w_spells.c w_quest.c w_dun.c
@@ -68,5 +68,5 @@ TARGET_LINK_LIBRARIES(tome lua ${LIBS})
 
 # Installation
 INSTALL(TARGETS tome
-  RUNTIME DESTINATION bin
+  RUNTIME DESTINATION games
 )
-- 
2.0.0.rc2

